client/tailwind.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 
     extend: {
       backgroundImage: {
-        'hero': 'url(../src/assets/portland_playhouse_cover.jpg)',
+        'hero': 'url(../src/assets/pexels-cottonbro-6896179.jpg)',
```

Comment:
I was thinking that we could make this even more generic. So something like a solid background color or a simple pattern that's on theme. Ideally Wondertix will be general enough that it wont be restricted to servicing playhouses. I also don't know there are copy right issues with this image?

Reply:
The image I placed there is Creative Commons so it is free to use in any way. I can switch to something more generic though